universal-application-tool-0.0.1/app/services/program/ProgramServiceImpl.java

```diff
@@ -115,7 +116,11 @@ public ProgramDefinition getProgramDefinition(long id) throws ProgramNotFoundExc
       return ErrorAnd.error(errors);
     }
     Program program =
-        programDefinition.toBuilder().setName(name).setDescription(description).build().toProgram();
+        programDefinition.toBuilder()
+            .setName(name)
+            .setDescription(description)
+            .build()
+            .toProgram(LifecycleStage.DRAFT);
```

Comment:
Question (non-blocking): I am curious why we are not adding stages when we create new programs.